### DISCUSSION TITLE
Profile Mode - Disable automatic uppercase button

### DISCUSSION THREAD

**Discussion by x1mus** [adityatelange/hugo-PaperMod#842] [Q&A] [answered]
Hello,
I'm kinda new to hugo and PaperMod in general and I would like to know if it is possible to disable the automatic uppercase modification of the first letter in buttons on the main page when we choosed the profile mod. (Of course, typing it in lower case doesn't work)
![image](https://user-images.githubusercontent.com/26910323/159161323-ec480a2d-028b-4ca4-976c-42c26e6a0b10.png)

I tried making some google research, some 'grep' on the display of every file but couldn't find anything
Thanks !

**Comment by x1mus** [comment, marked as answer]
I'm still looking for a solution if anyone has one!

**Reply by adityatelange** [reply]
I'll disable automatic upper case this weekend

**Reply by adityatelange** [reply]
It was been added as a feature to let people not worry about case of buttons, but nevertheless they can capitalise it if they want it that way.
